_o_components/02-o-combo.component.md

```diff
@@ -157,6 +157,19 @@ You can check a working example of this renderer [here](https://try.imatia.com/o
 
 You can check a working example of this renderer [here](https://try.imatia.com/ontimizeweb/v8/playground/main/inputs/combo).
 
+
+**Icon combo renderer**
+
+```html
+<o-combo attr="combo-editable-search" [static-data]="dataArray" [data]="valueSimple"
+  value-column="key" columns="key;value" visible-columns="value" required="yes" read-only="no" null-selection="no" searchable="yes" fxFlex>
+  <o-combo-renderer-icon icon-position="right" icon-column="materialIcon"></o-combo-renderer-percentage>
```

Comment:
I think it is necessary to change this example. I cannot see _materialIcon_ column defined in _columns_ attribute. And for better comprehension I think it should be great by using more real column names into configuration instead _key_ and _value_